src/inference/src/dev/plugin.hpp

```diff
@@ -59,6 +59,13 @@ class Plugin {
                                            const ov::SoPtr<ov::IRemoteContext>& context,
                                            const ov::AnyMap& config) const;
 
+    SoPtr<ov::ICompiledModel> import_model(std::istream& model, std::shared_ptr<ov::AlignedBuffer> model_buffer, const ov::AnyMap& properties) const;
+
+    SoPtr<ov::ICompiledModel> import_model(std::istream& model,
+                                           std::shared_ptr<ov::AlignedBuffer> model_buffer,
+                                           const ov::SoPtr<ov::IRemoteContext>& context,
+                                           const ov::AnyMap& config) const;
```

Comment:
```suggestion
    SoPtr<ov::ICompiledModel> import_model(std::istream& model,                                           
                                           const ov::SoPtr<ov::IRemoteContext>& context,
                                           const ov::AnyMap& config,
                                           std::shared_ptr<ov::AlignedBuffer> model_buffer = {}) const;
```

In usage it looks like optional.

Reply:
I prefer do not change existing API. It will also requires changes for iplugin and for all plugins impl